### PR TITLE
Environment variables for BinTray added

### PR DESCRIPTION
**Pull request by kbayraktar**
Gradle SHASUM version updated

### CODE REVIEW

**Comment by kbayraktar** [comment]
Please delete the environment variables from repository settings on travis before merging this branch into master !

**Comment by kbayraktar** [comment]
Please merge this project not until you delete the old repository environment variables settings on travis.

File: .travis.yml [hunk 1]
```diff
@@ -7,14 +7,21 @@ cache:
   - "$HOME/.gradle/wrapper/"
 before_install:
 - sudo mkdir -p /etc/nix
-- echo "substituters = https://cache.nixos.org/ file://$HOME/nix.store" | sudo tee -a /etc/nix/nix.conf > /dev/null
+- echo "substituters = https://cache.nixos.org/ file://$HOME/nix.store" | sudo tee
+  -a /etc/nix/nix.conf > /dev/null
```
Comment (jjelschen) [inline .travis.yml]: accidental line break…?

**Review by jjelschen — CHANGES_REQUESTED** [review]
please double-check, there's a newline that I think will break the script.

**Review by jjelschen — APPROVED** [review]

**Pull request merged**